com.unity.netcode.gameobjects/Editor/CodeGen/NetworkBehaviourILPP.cs

```diff
@@ -496,6 +499,13 @@ private bool ImportReferences(ModuleDefinition moduleDefinition)
                 }
             }
 
+
+            bool isEditor = assemblyDefines.Contains("UNITY_EDITOR");
```

Comment:
Ahh... I was wondering when you were going to use the assenblyDefines...and then it all made sense how you are fixing the exception and the visibility issue together.
🥇